src/tests/ftest/cart/iv_server.c

```diff
@@ -51,6 +52,9 @@ static int namespace_attached;
 
 static crt_group_t *grp;
 
+static uint32_t		g_grp_version;
+static int		g_timing;
```

Comment:
visual alignment.
Also we need to document here what values of 0, 1 and 2 mean and how they affect behavior